charts/internal/cilium/requirements.yaml

```diff
@@ -5,13 +5,9 @@ dependencies:
     condition: requirements.config.enabled
   - name: operator
     condition: requirements.operator.enabled
-  - name: preflight
-    condition: requirements.preflight.enabled
-  - name: nodeinit
-    condition: global.nodeinit.enabled
   - name: managed-etcd
     condition: global.etcd.managed
   - name: hubble-ui
     condition: requirements.hubble.enabled
   - name: hubble-relay
-    condition: requirements.hubble.enabled
```

Comment:
Nit: Changing the end-of-line was not really necessary, but does not hurt either.